components/scream/CMakeLists.txt

```diff
@@ -75,15 +84,15 @@ set(SCREAM_MPI_ERRORS_ARE_FATAL TRUE CACHE BOOL "Whether MPI errors should abort
 
 ## Work out pack sizes.
 # Determine the master pack size.
-set(SCREAM_PACK_SIZE ${DEFAULT_PACK_SIZE} CACHE INTEGER
+set(SCREAM_PACK_SIZE ${DEFAULT_PACK_SIZE} CACHE STRING
```

Comment:
Why did these integers become strings?

Reply:
Because cmake was giving me warnings about integers implicitly convertet to string. Cmake does not have 'INTEGER' for variable type (for some reason).

Reply:
You're right. The default is STRING and having it explicit here is confusing since these values are supposed to represent ints, so maybe just remove it?